clang/test/dpct/cache_remove.cu

```diff
@@ -0,0 +1,54 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none -out-root %T/cache_remove %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/cache_remove/cache_remove.dp.cpp --match-full-lines %s
+#include <cuda_runtime.h>
+
+__global__ void kernel()
+{
+  return;
+}
+
+int main()
+{
+  cudaStream_t stream;
+  const int pre_transform_tensor_size =
+      8 * 8 * 8;
+  const int transformed_tensor_size = pre_transform_tensor_size * 36 / 16;
+  const int res_block_mem =
+      transformed_tensor_size * 2 + pre_transform_tensor_size;
+  float *aptr;
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaLaunchAttributeValue is not supported.
```

Comment:
#define cudaStreamAttrValue cudaLaunchAttributeValue
In L21,  it's better to use cudaStreamAttrValue instead or else user will be mislead.

Reply:
fixed. now, we will use cudaStreamAttrValue in warning msg